File: LeetCode/May challenge/Day_10.cpp
class Solution {
public:
    int findJudge(int N, vector<vector<int>>& trust) {
        if(N==1)
            return 1;
        vector<int> a, b;
        for(int i=0; i<trust.size(); i++)
        {
            a.push_back(trust[i][0]);
            b.push_back(trust[i][1]);
        }
        for(auto i: b)
        {
            if(find(a.begin(),a.end(),i)==a.end() && count(b.begin(),b.end(),i)==N-1)
                return i;
        }
        return -1;
    }
};

File: LeetCode/May challenge/Day 10/Day_10.py
class Solution:
    def findJudge(self, N: int, trust: List[List[int]]) -> int:
        if N == 1:
            return 1
        trust_score = [0 for _ in range(N + 1)]
        
        for i, j in trust:
            trust_score[i] -= 1
            
            trust_score[j] += 1
            
        for i in range(1, N + 1):
            if trust_score[i] == N - 1:
                return i
        return -1
